dist/templates/banner-cookie-declaration-consent/banner-cookie-declaration-consent.html

```diff
@@ -95,17 +91,23 @@
       <div class="coi-consent-banner__categories-wrapper" id="coiConsentBannerCategoriesWrapper">
         {{#each cookie_categories}}
           <div class="coi-consent-banner__category-container">
-            <input class="coi-consent-banner__category-expander" id="coi-consent-banner__category-expander-{{this.cookie_type_label}}"
-              type="radio" name="tabs">
+            {{#if this.is_necessary}}
```

Comment:
You are not checking for unclassified category here, but you do below. Is this correct?

Reply:
Yes. It is.